tests/workflow/networking.tests.txt

```diff
@@ -32,25 +32,27 @@ eden.escript.test -test.run TestEdenScripts/eden_onboard
 eden.escript.test -test.run TestEdenScripts/template_check
 {{end}}
 
-/bin/echo Eden basic network test (5/{{$tests}})
-eden.escript.test -testdata ../network/testdata/ -test.run TestEdenScripts/network_test
+#/bin/echo Eden basic network test (5/{{$tests}})
+#eden.escript.test -testdata ../network/testdata/ -test.run TestEdenScripts/network_test
```

Comment:
They are not required anymore? Or it is some lines come from your debugging process?

Reply:
Good catch! I left it there accidentally.
It is fixed now.